Src/ElasticScale.Client/Microsoft.Azure.SqlDatabase.ElasticScale.Client.csproj

```diff
@@ -7,7 +7,7 @@
     <NeutralLanguage>en-US</NeutralLanguage>
     <Version>2.4.1</Version>
```

Comment:
>2.4.1 [](http://example.com/codeflow?start=13&length=5)

Rev this to 2.4.2-preview1 (can you check the syntax we used before, I think this is the way we do it)